src/transformers/configuration_pegasus.py

```diff
@@ -60,3 +60,46 @@ class PegasusConfig(BartConfig):
     @property
     def default_config_parameters(self):
         return DEFAULTS
+
```

Comment:
Don't think this has to be in config, no? Would prefer to have it only in the tests.